kamon-statsd/src/main/resources/reference.conf

```diff
@@ -5,18 +5,10 @@
 kamon {
   statsd {
 
-    # Hostname and port in which your StatsD is running. Remember that StatsD packets are sent using UDP and
-    # setting unreachable hosts and/or not open ports wont be warned by the Kamon, your data wont go anywhere.
-    hostname = "127.0.0.1"
-    port = 8125
```

Comment:
hey @ivantopo, although here both of the senders use single host/port pair, custom senders (like we do have) can use some kind of sharding technic to send to statsd to different hosts. So i believe it make sense to have host/port in sender specific settings.
for example my custom sender has this key instead of host and port.

```
    cluster-nodes = [
      "localhost:8125"
    ]
```
we can keep those for backward compatibility, but again i thinks this is up to sender
wdyt?